site/docs/transform/window.md

```diff
@@ -41,19 +46,24 @@ The window transform performs calculations over sorted groups of data objects. T
 
 ## Window Transform Definition
 
+Note: First, the tuples are partitioned according to the groupby fields. Each partition is then sorted. Finally, the window calculations are performed over the sorted partitions.
```

Comment:
Writing-wise, it's weird to have "Note" at the beginning of a section.

Reply:
I agree Note is weird at the beginning, but I did want this to be highlighted at the start. I wasn't quite sure where to put this, so I put it at the very beginning now right after the description of the window transform and removed the "Note:".